doc/user_guide/user_guide.md

```diff
@@ -99,6 +100,39 @@ Rules:
 
 * The Maven file must contain all necessary plugins and settings. Please check a [template](templates/maven_central_release_pom_file_template.md).
 
+### Rules for Release on Exasol Community Portal
+
+* The project must contain a `community_portal_post_template.json` file in the root directory. This json file has a few required objects:
+
+1. Array of strings `tags`: one or more tag of the project to be added to the community post;
+1. String `project name`: this is how the project name will be displayed in the post header;
+1. String `project description`: a short description of the project. Please, use markdown in the description;
+
+An example of the file:
+
+```json
+{
+  "tags": [
+    "Release Droid Testing",
+    "Java Tools",
+    "Open Source",
+    "GitHub"
+  ],
+  "project name": "Testing Release Droid",
+  "project description": "This is a project that helps us testing the [Release Droid](https://github.com/exasol/release-droid)."
```

Comment:
README is not parsing-friendly and the pom is not language independent. Also, pom can't have links, I think.
And maybe you want to have a slightly different description of the project in the release announcement than in the README file. So I would still leave it in this config file. It has benefits:

1) It's very easy to parse and validate
2) It forces users to think carefully about the project description they want to put on the Community Portal as they will add the file before using the release droid
3) It allows having a different description in the release announcement and in the README file
4) It's language-independent

Reply:
Well, parsing readme is possible. It's not as nice as JSON but doable.
I think the duplication has a higher risk of getting outdated than the advantage of having two different descriptions. Why do we need different ones. Both face users. We address developers usually in a later section of the README.

Anyway. I think if you want to keep the description here, you should pick a different format than JSON.
It should be easy to copy the description (including code examples) from the readme to this config file.
The current version encourages users to only write a one-sentence description, which leads to boring articles.
The format should support formatting (like markdown) so that you can provide code example. 
Editing markdown nested in a strings with `\n` is really no fun.